pkgs/top-level/haskell-packages.nix

```diff
@@ -55,7 +55,7 @@ in {
     };
     ghc844 = callPackage ../development/compilers/ghc/8.4.4.nix {
       bootPkgs = packages.ghc822Binary;
-      inherit (buildPackages.python3Packages) sphinx;
+      sphinx = buildPackages.python3Packages.sphinx_1_7_9;
```

Comment:
```suggestion
      sphinx = buildPackages.python3Packages.sphinx.overridePythonAttrs (oldAttrs: rec {
        version = "1.7.9";
        src = oldAttrs.src.override {
          inherit version;
        };
      });
```

Reply:
@dotlambda oh, that's cool! I'll remove sphinx/1.7.9.nix and apply your suggestion.